datahub-web-react/src/app/shared/TagStyleEntity.tsx

```diff
@@ -378,10 +396,11 @@ export default function TagStyleEntity({ urn, useGetSearchResults = useWrappedSe
                                         onClick={() =>
                                             navigateToSearchUrl({
                                                 type: aggregation?.value as EntityType,
-                                                query:
+                                                filters:
                                                     aggregation?.value === EntityType.Dataset
-                                                        ? entityAndSchemaQuery
-                                                        : entityQuery,
+                                                        ? entityAndSchemaFilters
+                                                        : entityFilters,
+                                                unionType: UnionType.OR,
```

Comment:
since we're removing the `query` here, do we want to pass in a `*` query with these filters? i see in `navigateToSearchUrl` that if `query` is undefined, we end up just using an empty string as the `query` - does that work the same?

Reply:
It does!

Reply:
awesome